WooCommerce/Classes/ViewRelated/ReusableViews/TextViewTableViewCell.swift

```diff
@@ -1,14 +1,39 @@
 import UIKit
 import Gridicons
 
-class TextViewTableViewCell: UITableViewCell {
+/// A table view cell that containt an icon and a text view.
+///
+final class TextViewTableViewCell: UITableViewCell {
+    struct ViewModel {
+        var icon: UIImage? = nil
+        var iconAccessibilityLabel: String? = nil
+        var iconTint: UIColor? = nil
+        var text: String? = nil
+        var placeholder: String? = nil
+        var textViewMinimumHeight: CGFloat? = nil
+        var isScrollEnabled: Bool = true
+        var onTextChange: ((_ text: String) -> Void)? = nil
+        var onTextDidBeginEditing: (() -> Void)? = nil
+        var keyboardType: UIKeyboardType = .default
+        var style: Style = .body
+        var edgeInsets: UIEdgeInsets?
+    }
 
     @IBOutlet private weak var noteIconView: UIView!
-    @IBOutlet var noteIconButton: UIButton!
+    @IBOutlet private var noteIconButton: UIButton!
+
+    @IBOutlet private var noteTextView: EnhancedTextView!
+
+    /// Constraints
+    @IBOutlet private weak var textViewHeightConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var bottomContraint: NSLayoutConstraint!
+    @IBOutlet private weak var trailingContraint: NSLayoutConstraint!
+    @IBOutlet private weak var leadingContraint: NSLayoutConstraint!
+    @IBOutlet private weak var topContraint: NSLayoutConstraint!
 
-    @IBOutlet var noteTextView: EnhancedTextView!
+    private var viewModel: ViewModel?
```

Comment:
[Q] Is it needed to for us to store the view model? I don't think views should store this extra state, as it could get outdated easily. EG: internal UIKit state changes but the view model doesn't.